lgc/patch/Patch.cpp

```diff
@@ -76,6 +76,9 @@ opt<bool> DisablePatchOpt("disable-patch-opt", desc("Disable optimization for LL
 opt<bool> UseLlvmOpt("use-llvm-opt",
                      desc("Use LLVM's standard optimization set instead of the curated optimization set"), init(false));
 
+// Determine optimization level.
+opt<char> OptLevel("O", desc("Optimization level. [-O0, -O2, -Oc] (default = '-O2')"), Prefix, ZeroOrMore, init(' '));
+
```

Comment:
Could this be an enum instead? I wouldn't mind having to type a longer flag, e.g., `--opt=O0`.